invenio_communities/assets/semantic-ui/js/invenio_communities/settings/profile/DeleteCommunityModal.js

```diff
@@ -13,6 +13,7 @@ import PropTypes from "prop-types";
 import { Trans } from "react-i18next";
 import { communityErrorSerializer } from "../../api/serializers";
 import { ErrorMessage, withCancel } from "react-invenio-forms";
+import axios from "axios";
```

Comment:
Note: It is not very common that we use axios directly like this, it's usually behind some kind of wrapper like in [React-Invenio-Deposit](https://github.com/inveniosoftware/react-invenio-deposit/blob/master/src/lib/DepositApiClient.js#L30) or inside a search application (e.g. in communities search). 

Question: is the record count hookable to the community in the backend? If so, we could avoid making one request to the API just for this. Otherwise, ignore this comment :)